app/helpers/basespace_helper.rb

```diff
@@ -0,0 +1,122 @@
+require 'open-uri'
+
+BASESPACE_CURRENT_PROJECTS_URL = "https://api.basespace.illumina.com/v1pre3/users/current/projects".freeze
```

Comment:
it looks a little odd to have one endpoint from `v1pre3` and one from `v2`. Is there not a `v2/users`?

Reply:
Not that I can see :( I pinged our Illumina contact again. Will add a TODO for now.